src/seedu/addressbook/commands/DeleteExamCommand.java

```diff
@@ -26,16 +27,18 @@ public DeleteExamCommand() {
     }
 
     public DeleteExamCommand(int targetVisibleIndex) {
-        super(targetVisibleIndex);
+        super(-1, targetVisibleIndex);
     }
```

Comment:
Pretty potato coding to set it as -1 arbitrarily, especially from the external point of view, in the context of this class, there is no reference to what -1 could mean. Refer to my previous comment about subclassing, which allows (int) to mean targeExam and (int, int) to mean both if ya ever need it.

(Hmm perhaps we can agree on whether to use optionals for targetIndex just to have a clearer indication of having no targets)

Reply:
I've changed index to variables within the command itself.